timeseries-streaming/timeseries-java-applications/Examples/src/main/java/com/google/dataflow/sample/timeseriesflow/examples/fsi/forex/ForexCSVAdaptor.java

```diff
@@ -45,6 +45,7 @@
 
 import static jdk.nashorn.internal.objects.Global.print;
 
+// This class is used for backtesting or bootstrap in batch mode, we would normally implement for streaming workloads
```

Comment:
This class is used for backtesting or bootstrap in batch mode. The primary usage of the timeseries library being for live stream mode processing.